roles/parse_c2r_report/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Default conversion_inhibited to false
+  ansible.builtin.set_fact:
+    conversion_inhibited: false
+
+- name: Collect human readable report results
+  ansible.builtin.slurp:
+    src: "{{ result_filename }}"
+  register: results_txt
+
+- name: Collect JSON report results
+  ansible.builtin.slurp:
+    src: "{{ result_filename_json }}"
+  register: results_json
+
+- name: Parse report results
+  ansible.builtin.set_fact:
+    convert2rhel_report_txt: "{{ results_txt.content | b64decode | split('\n') }}"
+    convert2rhel_report_json: "{{ results_json.content | b64decode | from_json }}"
+
+- name: Check for SKIP, OVERRIDABLE, ERROR statuses
+  ansible.builtin.set_fact:
+    conversion_inhibited: true
+  when:
+      # All of these are unnaceptable based on https://github.com/oamg/convert2rhel-worker-scripts/blob/main/scripts/preconversion_assessment_script.py#L121
+    - "'SKIP' == item.result.level or 'SKIP' in item.msg_levels"
+    - "'OVERRIDABLE' == item.result.level or 'OVERRIDABLE' in item.msg_levels"
+    - "'ERROR' == item.result.level or 'ERROR' in item.msg_levels"
```

Comment:
It's an elegant solution, I like it! 

Just a question... Is there a way for us to skip this task if we detect a specific key inside the report json? Asking this because in the next convert2rhel version (v1.6) we will have a key that will detect and give us the correct information, so, in theory, this won't be needed. 

Another approach is: Let's leave it the way it is as it won't break with a new version, and we can increment later and use the new approach in a future release of this project. I will let the decision to you, whatever it seems more reasonable for you.

Reply:
Well I don't know what will be the format, but let's say there will be in each action a key that would say `inhibitor:true`, we could then change this with something like this?

```
- name: Check for inhibitor true in actions
  ansible.builtin.set_fact:
    conversion_inhibited: true
  when: item.inhibitor == true
  loop: "{{ convert2rhel_report_json.actions }}"
```

or if the key is on top level, we could just set the variable based on it

```
- name: Check if conversion inhibited
  ansible.builtin.set_fact:
    conversion_inhibited: convert2rhel_report_json.inhibited
```

Reply:
but as you said, I went with the easiest that would still work with new version :)